torchx/lib/torchx/backend.ex

```diff
@@ -979,6 +979,41 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  @impl true
+  def window_max(out, tensor, window_dims_tuple, opts) do
+    if Enum.any?(opts[:padding], fn conf -> conf |> Tuple.to_list() |> Enum.any?(&(&1 != 0)) end) do
```

Comment:
@josevalim @seanmor5 I'm going to need a little help regarding the padding config. It's not really clear to me what's the equivalence there.